Update supported OS in meta/main.yml

jtruestedt opened the pull request:
Hi,
as previously mentioned in https://github.com/theforeman/ansible-foreman_scap_client/pull/31#issuecomment-796595636 I wanted to update the meta/main.yml with the supported OS.

I did not only add Ubuntu 20.04, but also SLES 15 and Debian 10, as I have tested them. However, I only added a test for Ubuntu 20.04. 
I could add the test for Debian 10 as well (should be the same as for Ubuntu with different repo and different image) - but for SLES 15 I do not have a public available repository with the subscription-manager and the scap-client - as well as geerlingguys docker images do not offer a SLES-container.

If you prefer I can also remove the SLES from the meta-information.

xprazak2 commented:
Even thought the difference between ubuntu/debian tests would be minimal, it would still be good to test against debian if we declare it as a supported OS. 

Not having SLES in meta is probably a safer choice if we won't do the testing.

jtruestedt commented:
I removed SLES and added the Tests for Debian 10
this should be considered in https://github.com/theforeman/ansible-foreman_scap_client/pull/32 as we have a new test scenario

xprazak2 commented:
Thanks!
